workflow.rst

```diff
@@ -452,6 +444,146 @@ missing a title::
 
     The optional second argument of ``setBlocked()`` was introduced in Symfony 5.1.
 
+.. _workflow-usage-choose-events:
+
+Choosing which Events to dispatch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You are able to specify which events will be fired when performing each
+transition by passing an array of workflow events to the ``events_to_dispatch``
```

Comment:
before showing config code, i would describe briefly the 3 options available:
- null => all events
- [] => no events
- [...|...] => specific events